mono/mini/mini-x86.c

```diff
@@ -3468,7 +3468,7 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 				x86_fld80_membase (code, X86_ESP, 2);
 				x86_alu_reg_imm (code, X86_ADD, X86_ESP, 12);
 			} else {
-				x86_fld80_mem (code, mn);
+				x86_fld80_mem (code, (gsize)&mn);
```

Comment:
shouldn't it be `(gsize)mn`?

Reply:
Same thing. I thought this might be clearer, but same thing.

Reply:
ah arrays. nevermind 🙂

Reply:
Could probably be `const` too.